tslint.json

```diff
@@ -1,41 +1,23 @@
 {
-  "rulesDirectory": [
-    "node_modules/codelyzer"
-  ],
+  "rulesDirectory": ["node_modules/codelyzer"],
```

Comment:
It's better to do linting/formatting changes in a separate commit so it's easier to review.

Reply:
Fixed!